content/applications/sales/crm/track_leads/unattended_leads_report.rst

```diff
@@ -0,0 +1,165 @@
+=======================
+Unattended leads report
+=======================
+
+*Unattended leads* are leads that have scheduled activities that are either due or past due.
+Whenever an activity is scheduled, *Odoo* tracks the due date and sends email reminders to the users
+the activity is assigned to.
+
+An *unattended leads report* compiles all active leads in the pipeline with due or past due
+activities, allowing a sales manager to identify which opportunities require immediate attention.
+
+By pulling a daily unattended leads report, sales managers can remind their teams to address
+outstanding activities before they become past due, helping avoid neglected leads and reinforcing
+proactive behaviors in their salespeople.
+
+.. example::
+
+   A sales manager starts their day by pulling an unattended leads report, and upon switching to
+   list view, they see the following:
+
+   .. image:: unattended_leads_report/unattended-leads-example.png
+      :align: center
+      :alt: List view of a sample Unattended Leads report with the activities emphasized.
+
+  Their team member, Mitchell, has two leads in the Proposition stage with activities that are due.
+  The yellow :guilabel:`📞 (phone)` icon indicates that the `5 VP Chairs` lead has a phone call
+  activity scheduled for today. The red :guilabel:`✉️ (envelope)` icon indicates that the `Modern
+  Open Space` lead has an email activity scheduled that is past due.
+
+  Clicking on the `Modern Open Space` lead, the sales manager opens the record of the lead and
+  reviews the chatter. They see that the email was scheduled to be sent two days ago, but Mitchell
+  never marked this activity as done.
+
+  .. image:: unattended_leads_report/overdue-activities-email.png
+     :align: center
+     :alt: Example of overdue activities notification in the chatter of a lead.
+
+.. important::
+   In order to pull an unattended leads report, activity track **must** be utilized in the *CRM*
+   pipeline.
+
+   For more information, refer to :ref:`Activity Tracking
+   <track_leads/unattended_leads_report/activity tracking>`
+
+Create an unattended leads report
+=================================
+
+To create an unattended leads report, first navigate to :menuselection:`CRM app --> Reporting -->
+Pipeline` to open the :guilabel:`Pipeline Analysis` report. Click into the search bar at the top of
+the page. The :guilabel:`Created on` filter can remain active, though if any other filters are
+present, they should be removed.
```

Comment:
- It's not *quite* a report yet (it's a *dashboard* where reports are generated)
- let's label the search bar with a gui and the placeholder text it's been given since it's now part of an active instruction.
- add transitional language
- wording adjustment

```suggestion
Pipeline` to open the :guilabel:`Pipeline Analysis` dashboard. Then, click into the
:guilabel:`Search...` bar at the top of the page, and remove all of the default filters. Note, the
:guilabel:`Created on` filter may remain active, as this variable can be useful to include in the
report.
```